apisupport/maven.apisupport/src/org/netbeans/modules/maven/apisupport/Bundle.properties

```diff
@@ -45,4 +45,6 @@ NbmWizardPanelVisual.cbAddModule.text=Create &Module Project:
 NbmWizardPanelVisual.cbAddModule.accessibledesc=Select to create a module project.
 NbmWizardPanelVisual.txtAddModule.accessiblename=module project name
 NbmWizardPanelVisual.txtAddModule.accessibledesc=Enter module project name.
-TXT_ProjectXmlFileElementValueRename=Update Element: {0} Value: {1}
\ No newline at end of file
+TXT_ProjectXmlFileElementValueRename=Update Element: {0} Value: {1}
+NbmWizardIterator.AddOpenFlag=@@metabuild.jms-flags@@
+NbmWizardIterator.AddOpenFFlag=@@metabuild.jms-fflags@@
```

Comment:
do we really need two properties? Is there nothing which could add the `-J` for the mavenproject.conf?

I assume the updated archetype will insert the properties to surefire configuration/argline etc?

Reply:
just a thought: would it be possible to make the launcher smarter so that it could use something like:
-J'-flag1 -flag2 -flag3' or -JVM_FLAGS='-flag1 -flag2 -flag3'?

Reply:
It's a more heavy change, for windows it would be there https://github.com/apache/netbeans-native-launchers unsure on how it works on mac